src/main/kotlin/com/ajaxproject/warehouse/dto/CustomerDataLiteDto.kt

```diff
@@ -1,10 +1,10 @@
 package com.ajaxproject.warehouse.dto
 
 data class CustomerDataLiteDto(
-    val id: Int?,
+    val id: String,
     val firstName: String,
     val surname: String,
-    val patronymic: String?,
+    val patronymic: String? = null,
     val email: String,
-    val phoneNumber: String?,
+    val phoneNumber: String? = null
```

Comment:
for some DTOs you declared the default value, for some you don't.

Let's be consistent and use the same practice for each DTO